src/services/AssetService.ts

```diff
@@ -6,17 +7,43 @@ import { FlowComponents } from '../store/helpers';
 export class Assets {
     private endpoint: string;
     private localStorage: boolean;
-    protected items: SearchResult[] = [];
     protected nameProperty: string;
     protected idProperty: string;
 
+    // TODO: consider redux backed by local storage for this
+    protected items: SearchResult[] = [];
```

Comment:
Is `objects` only going to be used for flows?

Reply:
admittedly terrible names for both to these... will come up with something better as I move forward on this.